### PR TITLE
Issue #401: Change package description

### LINKED ISSUE #401: Update description while we have chance
Right now we have: "Estimate epidemiological delay distributions for infectious diseases"

On the RHS we have: An R package for estimating epidemiological delay distributions 

I'd say "epidemiological delay distributions for infectious diseases" is pretty good / central though it does narrow us more than we actually are (it's not just epidemiological). Also, do we need to say "for infectious diseases"? What about non infectious diseases? Nothing about the methods is infectious specific?

Should we say with `brms`? I think we should. Maybe "Stan" has more name recognition?

Should we say it's an R package? Perhaps we don't need to say this.

### PR DESCRIPTION
**Pull request by athowes**
## Description

This PR closes #401. Self-explanatory.

## Checklist

- [x] My PR is based on a package issue and I have explicitly linked it.
- [x] I have included the target issue or issues in the PR title in the for Issue(s) *issue-numbers*: PR title
- [x] I have read the [contribution guidelines](https://github.com/epinowcast/.github/blob/main/CONTRIBUTING.md).
- [x] I have tested my changes locally.
- [x] I have added or updated unit tests where necessary.
- [x] I have updated the documentation if required.
- [x] My code follows the established coding standards.
- [ ] I have added a news item linked to this PR.
- [x] I have reviewed CI checks for this PR and addressed them as far as I am able.

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Package: epidist
-Title: Estimate epidemiological delay distributions for infectious diseases 
+Title: Estimate epidemiological delay distributions with brms
```
Comment (jamesmbaazam) [inline DESCRIPTION:2]: Just a drive-by review


Title should be capitalised. See [here](https://r-pkgs.org/description.html).

Comment (jamesmbaazam) [inline DESCRIPTION:2]: Just a drive-by review

Title should be capitalised. See here.

```suggestion
Title: Estimate Epidemiological Delay Distributions With brms 
```

Comment (athowes) [reply DESCRIPTION:2]: Thanks!

**Review by seabbs — APPROVED** [review]
